static_analyzer/tree_sitter_parsers.py

```
@@ -0,0 +1,267 @@
+        if language == Language.PHP:
+            import tree_sitter_php
+
+            return tree_sitter_php.language_php()
+    except Exception:
+        logger.debug("Could not load tree-sitter grammar for %s", language, exc_info=True)
+    return None
+
+
+def _get_old_content(repo_dir: Path, base_ref: str, file_path: str) -> str | None:
+    try:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)